Sources/PointFree/Emails/EmailLayouts.swift

```diff
@@ -4,6 +4,20 @@ import HtmlCssSupport
 import Prelude
 import Styleguide
 
+enum EmailLayoutTemplate {
```

Comment:
wanted to reuse the email layout for the blog, so made a way to support multiple templates.